docs/core/porting/index.md

```diff
@@ -41,7 +41,7 @@ For more information, see [Use the Windows Compatibility Pack to port code to .N
 
 ## .NET Framework compatibility mode
 
-The .NET Framework compatibility mode was introduced in .NET Standard 2.0. This compatibility mode allows .NET Standard and .NET 5+ (and .NET Core 3.1) projects to reference .NET Framework libraries on Windows-only. Referencing .NET Framework libraries doesn't work for all projects, such as if the library uses Windows Presentation Foundation (WPF) APIs, but it does unblock many porting scenarios. For more information, see the [Analyze your dependencies to port code from .NET Framework to .NET](third-party-deps.md#net-framework-compatibility-mode).
+The .NET Framework compatibility mode was introduced in .NET Standard 2.0. This compatibility mode allows .NET Standard and .NET (since .NET Core 3.1) projects to reference .NET Framework libraries on Windows-only. Referencing .NET Framework libraries doesn't work for all projects, such as if the library uses Windows Presentation Foundation (WPF) APIs, but it does unblock many porting scenarios. For more information, see the [Analyze your dependencies to port code from .NET Framework to .NET](third-party-deps.md#net-framework-compatibility-mode).
```

Comment:
- Option 1

This compatibility mode allows **.NET Standard, .NET Core and .NET** projects to reference .NET Framework libraries on Windows-only.

- Option 2
- 
This compatibility mode allows **.NET Standard, .NET Core 2.0+ and .NET** projects to reference .NET Framework libraries on Windows-only.

- Option 3

This compatibility mode allows **.NET Standard, .NET Core (since version 2.0) and .NET** projects to reference .NET Framework libraries on Windows-only.